independent-publisher-connectors/NFTMania/apiDefinition.swagger.json

```diff
@@ -0,0 +1,213 @@
+                  "description": "The transaction hash which is returned by the blockchain after minting.",
+                  "title": "transaction_hash",
+                  "x-ms-visibility": "important"
+                },
+                "transaction_external_url": {
+                  "type": "string",
+                  "description": "Transaction URL in an external blockchain explorer.",
+                  "title": "transaction_external_url",
+                  "x-ms-visibility": "important"
+                },
+                "mint_to_address": {
+                  "type": "string",
+                  "description": "Account address where the NFT was sent.",
+                  "title": "mint_to_address",
+                  "x-ms-visibility": "important"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "Name of the NFT.",
+                  "title": "name"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Description of the NFT.",
+                  "title": "description"
+                }
+              }
+            }
+          },
+          "402": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "response": {
+                  "type": "string",
+                  "description": "Response status either OK or NOK.",
+                  "title": "response"
+                },
+                "error": {
+                  "type": "object",
+                  "properties": {
+                    "status_code": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "HTTP response status code.",
+                      "title": "status_code",
+                      "x-ms-visibility": "important"
+                    },
+                    "code": {
+                      "type": "string",
+                      "description": "Machine readable code describing the error.",
+                      "title": "code",
+                      "x-ms-visibility": "important"
+                    },
+                    "message": {
+                      "type": "string",
+                      "description": "Detailed description of the error.",
+                      "title": "message",
+                      "x-ms-visibility": "important"
+                    }
+                  },
+                  "description": "error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Mint NFT",
+        "description": "Mints NFT using the image URL",
+        "operationId": "Authorization",
+        "parameters": [
+          {
+            "name": "API",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important"
```

Comment:
@shreyan1999 The 'x-ms-summary' property is required, and the 'description' property is required. please add both.